interactive_engine/executor/ir/core/src/glogue/extend_step.rs

```diff
@@ -48,6 +49,79 @@ impl ExactExtendEdge {
     pub fn get_direction(&self) -> PatternDirection {
         self.dir
     }
+
+    /// Use the ExactExtendEdge to generate corresponding edge_expand operator
+    pub fn generate_edge_expand(
+        &self, mut edge_opr: pb::EdgeExpand,
+    ) -> IrPatternResult<pb::logical_plan::Operator> {
+        // use start vertex id as tag
+        edge_opr.v_tag = Some((self.src_vertex_id as KeyId).into());
+        edge_opr.direction = self.dir as i32;
+        edge_opr.expand_opt = pb::edge_expand::ExpandOpt::Edge as i32;
+        Ok(edge_opr.into())
+    }
+
+    /// Use the ExactExtendEdge to generate corresponding path_expand related operators
+    /// Specifically, path_expand will be translated to:
+    /// If path_expand is the one to be intersected, translate path_expand(l,h) to path_expand(l-1, h-1) + endV() + edge_expand
```

Comment:
Can move this logic into physical layer.

Reply:
Fixed